source/_integrations/roborock.markdown

```diff
@@ -45,15 +45,17 @@ Once you log in with your Roborock account, the integration will automatically d
 
 Roborock devices have a variety of features that are supported on some devices but not on others. Only entities that your device supports will be added to your integration.
 
+As of Home Assistant 2023.12, as long as the cloud client is connected, we are able to instantly update some entities immediately, instead of waiting for the typical coordinator update. In the descriptions below, if an entity has a (🔥) by it, that means it will be automatically updated.
```

Comment:
I don't think this matters in general usage, I don't think we need to add this for that reason.

Integrations have lots of different ways to update information, but in the end, when using it, it doesn't matter.

Reply:
My only thought was that some. Are instant and some aren't. I could see users getting confused why their status updates as soon as they start a clean big the "in_cleaning" bool does not. But I'm not super attached and I'm gone with deleting this PR if you don't think it's needed

Reply:
Thought about it a little. I do see your point, but we don't add this information anywhere right now, making this PR an oddity.

Therefore, going to close this for now.

../Frenck